src/background/setToolbarIconFromTheme.test.ts

```diff
@@ -51,7 +51,7 @@ describe("setToolbarIconFromTheme", () => {
   // @ts-expect-error -- No need to mock the whole class for the test
   global.Image = class {
     src = "";
-    // eslint-disable-next-line no-restricted-syntax
+    // eslint-disable-next-line no-restricted-syntax -- This is a mock
     decode = jest.fn().mockResolvedValue(undefined);
```

Comment:
This comment shouldn't be there, the rule is specifically about mocks

<img width="734" alt="Screenshot 5" src="https://github.com/pixiebrix/pixiebrix-extension/assets/1402241/b3fcb462-9d22-44d9-82de-39912234bf67">

```suggestion
    decode = jest.fn();
```